subprojects/docs/src/samples/userguide/dependencyManagement/dependencyLocking/lockingSingleConfiguration/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+    java
+}
+
+repositories {
+    mavenCentral()
+}
+
+
+// tag::locking-one[]
+configurations {
+    "compileClasspath" {
```

Comment:
It doesn't compile with `compileClasspath`:

```
               ^ Unresolved reference. None of the following candidates is applicable because of receiver type mismatch: 
```

Reply:
Even though this isn't exactly the same, I'm going to suggest

```
configurations.getByName("compileClasspath") {
    resolutionStrategy.activateDependencyLocking()
}
```

Do you agree @eskatos?

Reply:
@eriwen, I prefer it the way it was, it's more close to the corresponding Groovy sample, making things easier to understand.

I opened https://github.com/gradle/kotlin-dsl/issues/1082 to look into why the generated accessors cannot be used with a configuration block.